_events/2018-12-14-galaxy-training-rostock.md

```diff
@@ -0,0 +1,48 @@
+---
+site: main
+tags: [training]
+title: Galaxy for linking Bisulfite sequencing with RNA sequencing 06.-08.03.2019 in Rostock
+starts: 2019-03-06
+ends: 2019-03-08
+organiser:
+  name: de.STAIR
+  email: destair@leibniz-fli.de
+location:
+  name: University of Rostock
+  street: Albert-Einstein-Str. 22
+  postal: 18059
+  city: Rostock
+  region: Mecklenburg-Vorpommern
+  country: Germany
+supporters:
+- denbi
+---
+
+Galaxy for linking Bisulfite sequencing with RNA sequencing, March 2019, registration is open!
+
+Contents:
```

Comment:
You can makes these markdown headers and it will render a bit more nicely. Just change to `## Contents` or so :)

Reply:
Right! It slipped my mind.